bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -49,6 +49,41 @@ limitations under the License.
   </dependencyManagement>
 
   <dependencies>
+    <!-- Beam Group: should come first since we will be running in the beam ecosystem -->
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>
+        beam-sdks-java-extensions-google-cloud-platform-core
+      </artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-runners-google-cloud-dataflow-java</artifactId>
+      <exclusions>
```

Comment:
mind adding a small comment about exclusions? At first glance I thought we were adding these as deps

Reply:
done. Thanks for the quick review